projects/igniteui-angular/src/lib/expansion-panel/expansion-panel.common.ts

```diff
@@ -23,11 +23,6 @@ export const IGX_EXPANSION_PANEL_COMPONENT = new InjectionToken<IgxExpansionPane
 
 export interface IExpansionPanelEventArgs extends IBaseEventArgs {
     event: Event;
-    /**
-     * @deprecated
-     * To get a reference to the panel, use `owner` instead.
-     */
-    panel?: IgxExpansionPanelBase;
```

Comment:
Same as `IgxMaskDirective.placeholder`, I think we should add a **breaking change** for this in the CHANGELOG